libmscore/measure.cpp

```diff
@@ -2884,7 +2884,8 @@ bool Measure::isEmpty(int staffIdx) const
 
 bool Measure::isCutawayClef(int staffIdx) const
       {
-      if (!isEmpty(staffIdx))
+      bool empty = (score()->staff(staffIdx)->cutaway() && isEmpty(staffIdx)) || !_mstaves[staffIdx]->visible();
```

Comment:
Would be perfect to have nullptr checks for score()->staff(staffIdx) and _mstaves[staffIdx]

Reply:
agreed, coming soon